Components/HeroPostHeader/Partials/Author/index.twig

```diff
@@ -0,0 +1,4 @@
+<div class="author">
+  <span class="author-label">Written by</span>
+  <a href="{{ post.author.link }}" class="author-link">{{ post.author }}</a>
```

Comment:
Are we adding schema.org attributes to our base components? Or at least `rel="author"` at a minimum for this link?

http://schema.org/author